### PR TITLE
Favorites db

### PR DESCRIPTION
**Pull request by DanielFadlon-Cherrypick**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -11,19 +12,42 @@ def create_app():
...
-    def main_screen():
+    def search_movie():
         if request.method == 'GET':
-            search_value = request.args.get('search_value')
-            movies = filter_movie_short(search_movies_by_title(search_value))
-            return json_response(movies)
+            search_value = request.args.get('value')
+            movies = search_movies_by_title(search_value)
+            if movies['Response'] == 'False':
+                return json_response([])
```
Comment (AlonShprung) [inline main.py]: Handle error appropriately

File: main.py [hunk 2]
```diff
@@ -11,19 +12,42 @@ def create_app():
...
+        if request.method == 'POST':
+            movie_id = request.args.get('movie_id')
+            movie = filter_movie_details(get_movie_by_id(movie_id))
+            add_favorite_movie(movie)
+            return json_response("SUCCESS")
+
+    @app.route('/favorites', methods=['GET'])
+    def get_favorites():
+        if request.method == 'GET':
+            return json_response(get_favorite_movies())
```
Comment (AlonShprung) [inline main.py]: Handle errors

File: main.py [hunk 3]
```diff
@@ -11,19 +12,42 @@ def create_app():
...
+
+    @app.route('/delete_movie_from_favorites', methods=['POST'])
+    def delete_movie_from_favorites():
+        if request.method == 'POST':
+            movie_id = request.args.get('movie_id')
+            delete_movie(movie_id)
+            return json_response("SUCCESS")
```
Comment (AlonShprung) [inline main.py:63]: Handle errors

**Review by AlonShprung — COMMENTED** [review]
Handle errors

File: main.py (hunk 1 above)
Comment (DanielFadlon-Cherrypick) [reply main.py]: returns empty list since there is not exists movies with the query name.. why I need o handle it differently ?

**Review by AlonShprung — APPROVED** [review]

**Pull request merged**